code/game/machinery/cryopod.dm

```diff
@@ -284,26 +284,25 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 		to_chat(user, span_notice("[src] is already occupied!"))
 		return
 
-	if(target.stat == DEAD)
-		to_chat(user, span_notice("Dead people can not be put into cryo."))
-		return
-
-	if(target.key && user != target)
-		if(iscyborg(target))
-			to_chat(user, span_danger("You can't put [target] into [src]. [target.p_theyre(capitalized = TRUE)] online."))
-		else
-			to_chat(user, span_danger("You can't put [target] into [src]. [target.p_theyre(capitalized = TRUE)] conscious."))
+	if(user != target)
```

Comment:
Make sure you modify and remove all code below this that checks for `user == target` as this will always be TRUE and `user != target` as the proc would have early returned if this was TRUE. You only need to do this if you had a reason for removing Cryoing of keyless mobs, otherwise I assume you'll revert your changes to this part of the code.